etl-data-pipeline/src/main/java/gov/cdc/etldatapipeline/changedata/repository/ProviderRepository.java

```diff
@@ -0,0 +1,15 @@
+package gov.cdc.etldatapipeline.changedata.repository;
+
+import gov.cdc.etldatapipeline.changedata.model.dto.Provider;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+
+@Repository
+public interface ProviderRepository extends JpaRepository<Provider, String> {
+    @Query(nativeQuery = true, value = "execute sp_D_PROVIDER_EVENT :person_uids")
```

Comment:
When I was testing out how to return the select statement, it looked like an additional parameter might be required to hold the returned object. Is that possible via this method?
Eg.
EXEC rdb_modern..sp_D_PATIENT_EVENT @user_id_list,
@patient_list = @patientListOp OUTPUT;

Reply:
Yeah I think it should be possible. 
`EXEC rdb_modern..sp_D_PATIENT_EVENT @user_id_list, @patientListOp OUTPUT`
Have to verify how it works with the output parameter in the repository service.

Reply:
I need to discuss this execution with Ravi because the output parameter does not seem straightforward. I will update you as I go.